server/log_rpc_server.go

```diff
@@ -431,9 +431,24 @@ func (t *TrillianLogRPCServer) GetLatestSignedLogRoot(ctx context.Context, req *
 		FirstTreeSize:  int64(req.FirstTreeSize),
 		SecondTreeSize: int64(root.TreeSize),
 	}
```

Comment:
Not sure it's still worth bundling things up into a `GetConsistencyProofRequest` -- the most important check (second < first) from `validateGetConsistencyProofRequest()` (which is the only thing that uses the request structure) is now covered below.

Reply:
`validatesGetConsistencyProofRequest` verifiers other components of the request, such as whether `first_tree_size > 0`.    Seems like it's nice for parity with GetConsistencyProof too.